onboarding-checklist.md

```diff
@@ -0,0 +1,140 @@
+# Onboarding Checklist #
+
+## Instructions ##
+
+When someone new joins the team:
+
+1. A team member adds the new team member to the `cisagov` organization in
+GitHub.
+1. Create a new issue called "Onboard _Name_".
+1. View the raw source of this file.
+1. Copy the text after the line below to the issue.
+1. Remove any block of tasks that doesn't apply to the new team member's role
```

Comment:
This is a complete sentence and could end in a period, like the other bullets in this block.

Reply:
Fixed in a848650149665216a068543dd1b50962d7ec2737